docs/source/imagequality.rst

```diff
@@ -0,0 +1,29 @@
+Image Quality features
```

Comment:
We need to add the new features' math description, too. (Math/focus_score.rst, Math/psd_slope.rst, etc)